packages/python/requirements.txt

```diff
@@ -1,4 +1,5 @@
 Django==4.2.16
 Flask==3.0.3
 requests==2.32.3
-Werkzeug==3.0.4
\ No newline at end of file
+Werkzeug==3.0.4
+uvicorn==0.32.0
```

Comment:
uvicorn is used for our example app and tests that are related to ASGI. So i think we should keep it so that we don't break ASGI support in the future